Dockerfile

```diff
@@ -6,6 +6,7 @@ FROM bash:${bashver}
 # context where it make sense to cite GNU Parallel).
 RUN apk add --no-cache parallel && \
```

Comment:
Please add `ncurses` after `parallel` so it's all in the same Docker image layer and remove the separate line 🙏

Reply:
Fixed in last commit.